core/src/apps/common/layout.py

```diff
@@ -6,6 +6,8 @@
 from trezor.ui.components.tt.text import Text
 from trezor.ui.container import Container
 from trezor.ui.qr import Qr
+from trezor.ui.scroll import Paginated
```

Comment:
istm this whole file should not be here?

Reply:
Yup, sorry.

Reply:
I think @mmilata has removed this in his changes.